src/components/AdditionalInformation.jsx

```diff
@@ -2,9 +2,9 @@ import React from "react";
 import {Form, Field, connect } from "formik";
 import ErrorMsg from "./ErrorMessage";
 import FormContainer from './FormContainer';
-import { formIncomplete } from "./checkFormCompletion";
+import { formIncomplete } from "../utilities/checkFormCompletion";
 import ButtonDisplay from "./buttonDisplay";
-import submitReport from "./submitReport";
+import submitReport from "../utilities/submitReport";
```

Comment:
Pretty much the same as above, but I might suggest a service here, since it's more business logic-y than generic.

```js
import { SubmitReport } from '../services/ReportService';
```

Not blocking

Reply:
justy fyi in the sample code you would need to change to a named export in this case.

Reply:
Created a service folder and moved